.chloggen/elasticsearchexporter_monotonic_delta_sum_gauge.yaml

```
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: elasticsearchexporter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note:  Map monotonic delta sum metric as gauge in dynamic templates
```

```suggestion
note:  Fix the warning on Kibana when using `sum` operation on delta temporality sum metric mapped as a counter
```

Focus a bit more on what it fixes for the user